src/matzah.py

```diff
@@ -122,11 +122,13 @@ def __init__(self, DEVICE, fcSize, numFC, styleDim, outputDim, numBlocks,
         self.Synthesis = SynthesisNetwork(DEVICE, outputDim,
                                           fcSize + numClasses, numBlocks,
                                           startDim, channels, kernel)
+        self.sigmoid = nn.Sigmoid()
 
     def forward(self, styleVec, classVec):
         latentStyle = self.Style.forward(styleVec, classVec)
         # now need to set up the other side of the network.
         generatedCurves = self.Synthesis.forward(latentStyle)
+        generatedCurves = 3 * self.sigmoid(generatedCurves) - 3
```

Comment:
This takes values on the interval [-3, 0]. Are we sure about this?

Reply:
Yeah this does not look quite right

Reply:
yeah should be 6 * self.sigmoid let me update.